sklearn/ensemble/_stacking.py

```diff
@@ -525,6 +545,10 @@ class StackingRegressor(RegressorMixin, _BaseStacking):
         `None` means 1 unless in a `joblib.parallel_backend` context. -1 means
         using all processors. See Glossary for more details.
 
+    passthrough : bool, default=False
+        Whether or not to concatenate the original data ``X`` with the output
```

Comment:
"When False, only the predictions of estimators will be used as training data for final_estimator. When True, the final estimator is trained on the predictions as well as the original training data."

WDYT?

Reply:
Sounds great, I adjusted the wording in both classifier and regressor docstrings.